sdk/maps/azure-maps-search/README.md

```diff
@@ -0,0 +1,168 @@
+# Azure Maps Search Package client library for Python
+
+This package contains a Python SDK for Azure Maps Services for Search.
+Read more about Azure Maps Services [here](https://docs.microsoft.com/azure/azure-maps/)
+
+[Source code](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/maps/azure-maps-search) | [API reference documentation](https://docs.microsoft.com/rest/api/maps/search) | [Product documentation](https://docs.microsoft.com/azure/azure-maps/)
+
+## _Disclaimer_
+
+_Azure SDK Python packages support for Python 2.7 has ended 01 January 2022. For more information and questions, please refer to <https://github.com/Azure/azure-sdk-for-python/issues/20691>_
+
+## Getting started
+
+### Prerequisites
+
+- Python 3.6 or later is required to use this package.
+- A deployed Maps Services resource. You can use the [Azure Portal](https://docs.microsoft.com/azure/azure-maps/quick-demo-map-app) to set it up.
+
+### Install the package
+
+Install the Azure Maps Service Search SDK.
+
+```bash
+pip install --pre azure-maps-search
+```
+
+### Authenticate the client
+
+In order to interact with the Document Translation feature service, you will need to create an instance of a client.
+An **endpoint** and **credential** are necessary to instantiate the client object.
+
+### Create the client with AzureKeyCredential
+
+To use an [API key] as the `credential` parameter,
+pass the key as a string into an instance of [AzureKeyCredential][azure-key-credential].
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.maps.search import SearchClient
+
+endpoint = "https://<resource-name>.mapsservices.azure.com/"
+credential = AzureKeyCredential("<api_key>")
+search_client = SearchClient(endpoint, credential)
+```
+
+### Create the client with an Azure Active Directory credential
+
+`AzureKeyCredential` authentication is used in the examples in this getting started guide, but you can also
+authenticate with Azure Active Directory using the [azure-identity][azure_identity] library.
+
+To use the [DefaultAzureCredential][default_azure_credential] type shown below, or other credential types provided
+with the Azure SDK, please install the `azure-identity` package:
+
+```pip install azure-identity```
+
+Once completed, set the values of the client ID, and client secret of the AAD application as environment variables:
+`AZURE_CLIENT_ID`, `AZURE_CLIENT_SECRET`.
+
+```python
+from azure.identity import DefaultAzureCredential
+from azure.maps.search import SearchClient
+credential = DefaultAzureCredential()
+client_id = os.environ.get("CLIENT_ID")
+authentication_policy = AzureKeyCredential(os.environ.get("SUBSCRIPTION_KEY"))
```

Comment:
It is weird to use `SUBSCRIPTION_KEY` here instead of `AZURE_CLIENT_SECRET`?

Reply:
I will keep it constant and use `SUBSCRIPTION_KEY` now